.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
```

Comment:
Not sure if we should force the templates. I know that many in the Gutenberg team prefer to not use them, so I'd lean towards leaving this as `true`.

Reply:
I've never actually noticed the Open a blank issue link before. I think it's fine to disable it.

Reply:
> I've never actually noticed the Open a blank issue link before. I think it's fine to disable it. I usually click the first template, select all and delete the text.

👍 yeah alright. It's easy enough to select all of the text and delete if you don't like the template, anyway.

> It's a shame there's no option for templates that are only visible to contributors (e.g. overview issues).

That would be really neat.

Reply:
```suggestion
blank_issues_enabled: false
```